client/packages/common/src/ui/forms/JsonForms/api/api.ts

```diff
@@ -47,3 +47,18 @@ export const getDocumentRegistryQueries = (sdk: Sdk) => ({
     },
   },
 });
+
+export const getAllocateNumberMutations = (sdk: Sdk, storeId: string) => ({
```

Comment:
this is a very specific version of a general helper. Myself, I would add an `allocate` node to `getDocumentRegistryQueries` though you may think that this is crossing boundaries of naming and implementation

```
export const getDocumentRegistryQueries = (sdk: Sdk) => ({
  get: {
    ...etc
  },
  allocate: {
    number: : async (numberName: string): Promise<number> => {
...etc

  },
});
```

alternatively, rename this method to `getMutations` or `getAllocateMutations` or `getAllocationMutations` .. but I don't think that's needed. I would put it on the document registry method. Will there be more helpers? If not, just a `getAllocateNumber` method

Reply:
the number allocation is unrelated to the document registry. I renamed getAllocateNumberMutations to getAllocateNumber though